extraction/image_extractor.py

```diff
@@ -0,0 +1,40 @@
+"""This script checks whether DOM has image tag or not and
+creates and returns the Image object"""
+
+from extraction.data_models.image import Image
+from extraction.interface_content_extractor import IContentExtractor
+
+
+class ImageExtractor(IContentExtractor):
+    """This class inherits IContentExtractor to extract Images"""
+
+    def get_size(self, node):
+        """Returns the size of the image"""
+
+        if node.has_attr('width') and node.has_attr('height'):
+            return node['width'], node['height']
+        return 0, 0
+
+    def get_image_attributes(self, node, image):
```

Comment:
Add _, and move it down.

Reply:
Converted to create_image and moved down.